drodrpg/Data/Help/1/script.html

```diff
@@ -311,6 +314,11 @@ <h3><u>Character Script Commands</u></h3>
 	  then it will show the room on the minimap as if the player has already
 	  explored it.  If the indicated room has already been explored
 	  by the player, this command does nothing.</li>
+  <li><a name="set-map-icon"><b>Set map icon</b></a> - Adds an icon for the selected
+      room on the player's map. Icons are only shown for rooms that have been revealed
```

Comment:
Suggest rephrasing for clarity:
Icons can be added to unrevealed rooms but are only displayed on rooms that have been revealed to the player.